ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+class ImagesListViewController: UIViewController {
+    
+    @IBOutlet private var tableView: UITableView!
+    
+    private let photosName: [String] = Array(0..<20).map{ "\($0)" }
```

Comment:
перед открывающейся фигурной всегда должен быть пробел)
```suggestion
    private let photosName: [String] = Array(0..<20).map { "\($0)" }
```

Reply:
Принял к сведению и поправил.